src/batched/sparse/impl/KokkosBatched_GMRES_Serial_Impl.hpp

```diff
@@ -54,7 +54,7 @@
 #include "KokkosBatched_Givens_Serial_Internal.hpp"
 #include "KokkosBatched_Trsm_Decl.hpp"
 #include "KokkosBatched_Identity.hpp"
-#include "KokkosBatched_Gemv_Decl.hpp"
+#include "KokkosBlas2_serial_gemv_impl.hpp"
```

Comment:
Why use impl header instead of public header?

Reply:
I tried to avoid pulling in unnecessary dependencies in case of implementation-to-implementation calls - like with `SerialGMRES` using `SerialGemv` here. Let me know if you prefer a public header in these cases anyway.